src/CodeStyle/Core/Analyzers/Options/OptionSetExtensions.cs

```diff
@@ -4,29 +4,29 @@
 
 namespace Microsoft.CodeAnalysis
 {
-    internal static class AnalyzerConfigOptionsExtensions
+    internal static class OptionSetExtensions
     {
-        public static T GetOption<T>(this OptionSet analyzerConfigOptions, Option<T> option)
+        public static T GetOption<T>(this OptionSet optionSet, Option<T> option)
         {
-            if (!TryGetEditorConfigOption(analyzerConfigOptions, option, out T value))
+            if (!TryGetEditorConfigOption(optionSet, option, out T value))
             {
                 value = option.DefaultValue;
             }
 
             return value;
         }
 
-        public static T GetOption<T>(this OptionSet analyzerConfigOptions, PerLanguageOption<T> option, string language)
+        public static T GetOption<T>(this OptionSet optionSet, PerLanguageOption<T> option, string language)
         {
-            if (!TryGetEditorConfigOption(analyzerConfigOptions, option, out T value))
+            if (!TryGetEditorConfigOption(optionSet, option, out T value))
             {
                 value = option.DefaultValue;
             }
 
             return value;
         }
 
-        private static bool TryGetEditorConfigOption<T>(this OptionSet analyzerConfigOptions, IOption option, out T value)
+        private static bool TryGetEditorConfigOption<T>(this OptionSet optionSet, IOption option, out T value)
```

Comment:
What does this method do that optionSet.GetOption<T> doesn't do directly? The walking to find the explicit EditorConfigStorageLocation omits the naming styles case...?

Reply:
(maybe my more general question is "who uses these methods?"?)

Reply:
Looks like most things in the `*.CodeStyle/Formatting` use these methods.

Reply:
Many files are shared between the Workspace and CodeStyle layers. These extension methods are to handle getting option values when being built as part of the CodeStyle layer and need to match the method names used in the Workspace layer.